_includes/home/connect.md

```diff
@@ -48,9 +48,9 @@ When you build on Firefox, you build with a community of extension developers. A
 <div class="grid-container grid-x grid-padding-x align-center tiles-footer">
 <div class="cell small-12 medium-8 xlarge-6 text-center" markdown="1">
 
-[Learn more about the community](https://wiki.mozilla.org/Add-ons/Community/)
+[Learn more about the community](/Community/)
 
-[Learn about enhancing the API](https://wiki.mozilla.org/Add-ons/Community/)
+[Learn about enhancing the API](https://extensionworkshop.com/community/#get-involved-in-the-community)
```

Comment:
This link needs the origin (`https://extensionworkshop.com`) to be removed since it's a link to this site, that way it will work in all envs rather than just production.